system/Debug/Toolbar.php

```diff
@@ -348,6 +348,9 @@ protected function roundTo(float $number, int $increments = 5): float
      *
      * @param RequestInterface  $request
      * @param ResponseInterface $response
+     *
+     * @return void
+     * @phpstan-return never|void
```

Comment:
Is there any `exit()` or `die()` in this method?

Reply:
There does not include exit() or die().
So, I changed there to `@phpstan-return void`.

Reply:
done.